demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -513,7 +514,9 @@ def _get_unmockable_tests_from_conf(self) -> list:
                                             integration_name in unmockable_integrations]
             if test_name and (not test_record.test_integrations or unmockable_integrations_used) or not self.isAMI:
                 unmockable_tests.append(test_record)
-
+                continue
+            if test_record.is_mockable is False:
```

Comment:
```suggestion
            if not test_record.is_mockable:
```

Reply:
@Itay4 I don't want this to affect the tests unless specifically given.
When not specifically given the value will be `None` and with your suggestion it will make all tests unmockable.